app/routes/core.rb

```diff
@@ -125,6 +125,10 @@ def tracks
           ExercismWeb::Presenters::Tracks.tracks
         end
 
+        def active_tracks
+          X::Track.all.select(&:active)
```

Comment:
I have a stupid question here. By using `@active_tracks ||=` are we caching the data across many requests or a single request. I tested and found that it sends a API request for every refresh. Am I doing something wrong ?